client/me/site-blocks/main.jsx

```diff
@@ -68,7 +68,7 @@ class SiteBlockList extends Component {
 				<PageViewTracker path="/me/site-blocks" title="Me > Blocked Sites" />
 				<DocumentHead title={ translate( 'Blocked Sites' ) } />
 				<MeSidebarNavigation />
-				<SectionHeader label={ translate( 'Blocked Sites' ) } />
+				<SectionHeader label={ translate( 'Blocked sites' ) } />
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.